unittests/tools/test_vcg_parser.py

```diff
@@ -202,22 +115,7 @@ def test_parseissuerow_with_row_has_finding(self):
 class TestVCGImport(DojoTestCase):
 
     def test_can_parse_xml(self):
-        content = """<?xml version="1.0" encoding="utf-8"?>
-        <!--XML Export of VCG Results for directory: C:\\Projects\\WebGoat.Net. Scanned for C# security issues.-->
-        <CodeIssueCollection>
-        <CodeIssue>
-        <Priority>6</Priority>
-        <Severity>Suspicious Comment</Severity>
-        <Title>Comment Indicates Potentially Unfinished Code</Title>
-        <Description>The comment includes some wording which indicates that the developer regards
-        it as unfinished or does not trust it to work correctly.</Description>
-        <FileName>Findings.xml</FileName>
-        <Line>21</Line>
-        <CodeLine>TODO: Check the Code</CodeLine>
-        <Checked>False</Checked>
-        <CheckColour>LawnGreen</CheckColour>
-        </CodeIssue>
-        </CodeIssueCollection>"""
+        content = open("unittests/scans/vcg/can_parse_xml.xml")
```

Comment:
Please wrap this in a `with` as done recently in #10055 to prevent unclosed file warnings